### PR TITLE
Update the Project Test Case 01

### PR DESCRIPTION
**Pull request by isuruudapadi**
Update the Project Test Case 01
Need Members has to check conflict with the current update of the Isuru Branch

### CODE REVIEW

File: src/library/payfine/PayFineControl.java [hunk 1]
```diff
@@ -4,63 +4,63 @@
...
-		this.Ui = uI;
-		uI.SeT_StAtE(PayFineUI.uI_sTaTe.READY);
-		StAtE = cOnTrOl_sTaTe.READY;		
+		this.ui = ui;
+		ui.SeT_StAtE(PayFineUI.uI_sTaTe.READY);
+		state = ControlState.READY;
 	}
 
 
 	public void CaRd_sWiPeD(int MeMbEr_Id) {
```
Comment (shamalrathnayaka) [inline src/library/payfine/PayFineControl.java:31]: Could you please change the method here

File: src/library/payfine/PayFineControl.java [hunk 2]
```diff
@@ -4,63 +4,63 @@
...
-		Ui.DiSplAY(MeMbEr.toString());
-		Ui.SeT_StAtE(PayFineUI.uI_sTaTe.PAYING);
-		StAtE = cOnTrOl_sTaTe.PAYING;
+		ui.DiSplAY(member.toString());
+		ui.SeT_StAtE(PayFineUI.uI_sTaTe.PAYING);
+		state = ControlState.PAYING;
 	}
 	
 	
 	public void CaNcEl() {
```
Comment (shamalrathnayaka) [inline src/library/payfine/PayFineControl.java:47]: Could you please change the method here

File: src/library/payfine/PayFineControl.java [hunk 3]
```diff
@@ -4,63 +4,63 @@
...
-		Ui.SeT_StAtE(PayFineUI.uI_sTaTe.CANCELLED);
-		StAtE = cOnTrOl_sTaTe.CANCELLED;
+		ui.SeT_StAtE(PayFineUI.uI_sTaTe.CANCELLED);
+		state = ControlState.CANCELLED;
 	}
 
 
 	public double PaY_FiNe(double AmOuNt) {
```
Comment (shamalrathnayaka) [inline src/library/payfine/PayFineControl.java:53]: Could you please change the method here

File: src/library/payfine/PayFineControl.java [hunk 4]
```diff
@@ -4,63 +4,63 @@
...
 			throw new RuntimeException("PayFineControl: cannot call payFine except in PAYING state");
 			
-		double ChAnGe = MeMbEr.payFine(AmOuNt);
-		if (ChAnGe > 0) 
-			Ui.DiSplAY(String.format("Change: $%.2f", ChAnGe));
-		
-		Ui.DiSplAY(MeMbEr.toString());
-		Ui.SeT_StAtE(PayFineUI.uI_sTaTe.COMPLETED);
-		StAtE = cOnTrOl_sTaTe.COMPLETED;
+		double ChAnGe = member.payFine(AmOuNt);
```
Comment (shamalrathnayaka) [inline src/library/payfine/PayFineControl.java:57]: Could you please change the variable here

File: src/library/payfine/PayFineUI.java [hunk 5]
```diff
@@ -89,11 +90,13 @@ private String input(String prompt) {
 		
 		
 	private void Output(Object object) {
```
Comment (shamalrathnayaka) [inline src/library/payfine/PayFineUI.java:92]: Could you please change the method here and the others as well

**Comment by viboosha** [comment]
@isuruudapadi  can you please do the changes

**Pull request merged**